Update deploy-configure-credentials.md (#115049)
Azure/azure-content: articles/app-service/deploy-configure-credentials.md

```azurecli-interactive
az webapp deployment list-publishing-credentials --resource-group <group-name> --name <app-name> --query scmUri
```

> Note that the Git remote URI returned will not contain `/<app-name>.git` at the end. Ensure that you append `/<app-name>.git` when adding the remote URI, to avoid an error 22 with git-http-push. Additionally, when using `git remote add ... ` via shells that use the dollar sign for variable interpolation (such as bash), it's important to escape any dollar signs (`\$`) in the username and/or password. Failure to do so can result in authentication errors.

Reviewer: ```suggestion
Note that the returned Git remote URI doesn't contain `/<app-name>.git` at the end. When you add the remote URI, make sure to append `/<app-name>.git` to avoid an error 22 with `git-http-push`. Additionally, when using `git remote add ... ` via shells that use the dollar sign for variable interpolation (such as bash), escape any dollar signs (`\$`) in the username or password. Failure to escape this character can result in authentication errors.
```
Minor edits for Microsoft style
